examples/api/checkout.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const express = require('express');
+const multer = require('multer');
+const upload = multer();
+const {setMaxAge} = require('@lib/utils/cacheHelpers');
+
+// eslint-disable-next-line new-cap
+const examples = express.Router();
+const SHOPPING_CART_TOTAL = 9.94;
+const discounts = [];
```

Comment:
Can you use a cache with a max-age of 10min, I think LRU cache which we have as a dependency supports it (I know we haven't had it before, but we should have ;-)).

Reply:
exactly

Reply:
PTAL